lib/resources/r4/charge_item_create.yaml

```diff
@@ -0,0 +1,426 @@
+---
```

Comment:
Can you please review the structure and notes that have been done on the Modify feature fhir.cerner.com documentation to make sure the create is also in sync? https://github.com/cerner/fhir.cerner.com/pull/780/files?authenticity_token=uwEEH%2Bfxifl1rPvDH1HnciXQIox4yzgSotSahdkV2UUsj0ZFptY0ay1zE%2BRIEmy16hyXh%2BdzUH%2BM%2Be8kvpKvdg%3D%3D&file-filters%5B%5D=.rb&file-filters%5B%5D=.yaml#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938

Reply:
Both synced to similar https://github.com/cerner/fhir.cerner.com/pull/778/files#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR2-R3